yellow-paper/docs/public-vm/avm-circuit.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 1
+---
+
+# AVM Circuit
+
+## Memory
+**VM memory** refers to the VM circuit's top-level memory structure. Since the VM circuit executes an entire public execution request, VM memory contains a dedicated region for each of a request's message calls.
```

Comment:
@dbanks12 Is there a typo at the end of the sentence? (singular vs plural): "for each of a request's message calls"

Reply:
I do not believe so.

Reply:
But maybe I can simplify this sentence.